pom.xml

```diff
@@ -40,7 +40,7 @@
         <dependency>
             <groupId>com.github.ONSdigital</groupId>
             <artifactId>dp-dd-backend-model</artifactId>
-            <version>develop-SNAPSHOT</version>
+            <version>feature~metadata-SNAPSHOT</version>
```

Comment:
If I merge the model first then the existing API and DB loader projects will break until they are updated. On the other hand, I can't merge those projects pointing at develop-SNAPSHOT as that will still be the old version. So we either accept that some things will be broken for a short time, or merge them first pointing at the feature branch, then merge the model, and then update them to point at the develop version of the model again.